sympy/stats/tests/test_continuous_rv.py

```diff
@@ -709,7 +750,34 @@ def test_precomputed_cdf():
         compdiff = simplify(compdiff.rewrite(erfc))
         assert compdiff == 0
 
+
+def test_long_precomputed_cdf():
+    x = symbols("x", real=True, finite=True)
+    distribs = [
+            Arcsin("A", -5, 9),
+            Dagum("D", 4, 10, 3),
+            Erlang("E", 14, 5),
+            Frechet("F", 2, 6, -3),
+            Gamma("G", 2, 7),
+            GammaInverse("GI", 3, 5),
+            Kumaraswamy("K", 6, 8),
+            Laplace("LA", -5, 4),
+            Logistic("L", -6, 7),
+            Nakagami("N", 2, 7),
+            StudentT("S", 4)
+            ]
+    for distr in distribs:
+        for _ in range(5):
+            assert abs((simplify(diff(cdf(distr)(x), x))
```

Comment:
You probably don't need simplify here, since you are just plugging in a number for floating point evaluation. simplify is slow. Also, you don't even need all of the import random thing: there is `verify_numerically` method which takes care of things. See an example [here](https://github.com/sympy/sympy/pull/13909). Use it with `a=0, b=0, c=1, d=0` and you'll get verification for random numbers between 0 and 1. 

E.g., `assert tn(diff(cdf(distr)(x), x), density(distr)(x), x, a=0, b=0, c=1, d=0)`. It's much cleaner. 

Same for UniformSum cdf below.